### ISSUE TITLE
[Bug]: deny to insert after upgrade from master-20221027-d5bc4e25 to master-20221028-89541a6f

### ISSUE THREAD

**Issue by yanliang567** [milvus-io/milvus#20163]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:  master-20221028-89541a6f
- Deployment mode(standalone or cluster): cluster
- SDK version(e.g. pymilvus v2.0.0rc2):
```

### Current Behavior

it shows 
`force to deny /milvus.proto.milvus.MilvusService/Insert`
`force to deny /milvus.proto.milvus.MilvusService/Import` 
after upgrade from master-20221027-d5bc4e25 to master-20221028-89541a6f

### Expected Behavior

no limit or deny
[yanliang-bulkload-milvus-proxy-76d8ddd77-zfcvq.log](https://github.com/milvus-io/milvus/files/9886124/yanliang-bulkload-milvus-proxy-76d8ddd77-zfcvq.log)
[yanliang-bulkload-milvus-mixcoord-54687cbdd4-pw7qh.log](https://github.com/milvus-io/milvus/files/9886127/yanliang-bulkload-milvus-mixcoord-54687cbdd4-pw7qh.log)
[yanliang-bulkload-milvus-datanode-84c49859b6-rzk72.log](https://github.com/milvus-io/milvus/files/9886190/yanliang-bulkload-milvus-datanode-84c49859b6-rzk72.log)

### Steps To Reproduce

```markdown
1. run test_e2e.py and some bulk_insert tests
2. upgrade milvus 
3. run test_e2e or bulk insert tests again after all pods running
```

**Comment by yanliang567** [comment]
last time when trying to upgrade from a dev image to master-20221027-d5bc4e25 , the data node crashed for times
[milvus-log.tar.gz](https://github.com/milvus-io/milvus/files/9886198/milvus-log.tar.gz)

**Comment by bigsheeper** [comment]
Datanode seek from a very early position:
```
[2022/10/28 02:53:07.868 +00:00] [DEBUG] [datanode/flow_graph_insert_buffer_node.go:91] ["IBN timetick log"] [from=2022/10/27 08:28:29.061 +00:00] [to=2022/10/27 08:31:49.010 +00:00] [elapsed=3m19.949s] [start=436957166714486785] [end=436957219129917441] [vChannelName=yanliang-bulkload-rootcoord-dml_2_436956674908163471v0]
```

**Comment by bigsheeper** [comment]
Collection create about 18h ago:
![image](https://user-images.githubusercontent.com/42060877/199427685-c3358d12-1f57-4057-8c6e-97f663fe9df0.png)

and no any insert operation:
![image](https://user-images.githubusercontent.com/42060877/199428686-ec5f2395-4b89-4896-9f55-4bdf35af18c2.png)

So after upgrade and datanode restarted, flowgraphs consumes from cp of 18h ago. So time tick long delay.

**Comment by bigsheeper** [comment]
Which is same as https://github.com/milvus-io/milvus/issues/17445

**Comment by bigsheeper** [comment]
should be fixed @yanliang567

**Comment by yanliang567** [comment]
verified fixed on v2.2.0
